editor/script/room.js

```diff
@@ -195,6 +195,10 @@ function RoomTool(canvas) {
 		isDragDeletingTiles = false;
 	}
 
+	function clamp(val, min, max){
```

Comment:
nice work!

this is a nitpick (and I realize there isn't a very consistent code style in bitsy D:) but I like to put a space before the opening parentheses of a function definition :)

otherwise, this looks great!